[2025-05-02T22:37:25.626+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=statsbomb_full_ingestion/run_id=manual__2025-05-02T22:36:19.789358+00:00/task_id=ingest_events_lineups permission to 509
[2025-05-02T22:37:25.694+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=statsbomb_full_ingestion/run_id=manual__2025-05-02T22:36:19.789358+00:00/task_id=ingest_events_lineups permission to 509
[2025-05-02T22:37:25.757+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statsbomb_full_ingestion.ingest_events_lineups manual__2025-05-02T22:36:19.789358+00:00 [queued]>
[2025-05-02T22:37:25.775+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statsbomb_full_ingestion.ingest_events_lineups manual__2025-05-02T22:36:19.789358+00:00 [queued]>
[2025-05-02T22:37:25.777+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-05-02T22:37:25.803+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): ingest_events_lineups> on 2025-05-02 22:36:19.789358+00:00
[2025-05-02T22:37:25.811+0000] {standard_task_runner.py:57} INFO - Started process 1225 to run task
[2025-05-02T22:37:25.815+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'statsbomb_full_ingestion', 'ingest_events_lineups', 'manual__2025-05-02T22:36:19.789358+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/statsbomb_ingestion.py', '--cfg-path', '/tmp/tmpgi99j8ui']
[2025-05-02T22:37:25.821+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask ingest_events_lineups
[2025-05-02T22:37:25.915+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=statsbomb_full_ingestion/run_id=manual__2025-05-02T22:36:19.789358+00:00/task_id=ingest_events_lineups permission to 509
[2025-05-02T22:37:25.935+0000] {task_command.py:415} INFO - Running <TaskInstance: statsbomb_full_ingestion.ingest_events_lineups manual__2025-05-02T22:36:19.789358+00:00 [running]> on host e2d1066c2b98
[2025-05-02T22:37:26.081+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='statsbomb_full_ingestion' AIRFLOW_CTX_TASK_ID='ingest_events_lineups' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T22:36:19.789358+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T22:36:19.789358+00:00'
[2025-05-02T22:37:26.476+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/statsbomb_ingestion.py", line 79, in ingest_events_lineups
    for match in comp["matches"]:
KeyError: 'matches'
[2025-05-02T22:37:26.529+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=statsbomb_full_ingestion, task_id=ingest_events_lineups, execution_date=20250502T223619, start_date=20250502T223725, end_date=20250502T223726
[2025-05-02T22:37:26.558+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 22 for task ingest_events_lineups ('matches'; 1225)
[2025-05-02T22:37:26.577+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-02T22:37:26.649+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
